SConstruct

```diff
@@ -172,6 +175,19 @@ def render_thumbnail(env, target, source):
                                                  resolution=env['RESOLUTION'],
                                                  appleseed_path=APPLESEED_PATH)
 
+    else:
+        process = batchtools.sbsrender_render(
+            os.path.abspath("./data/dependencies/pbr_render.sbsar"),
+            output_name=os.path.splitext(os.path.basename(str(target[0])))[0],
+            output_path=os.path.dirname(str(target[0])),
+            engine="ogl3" if platform.system() == "Linux" else "d3d10pc",
+            set_entry=["basecolor@{}".format(str(source[0])),
+                       "metallic@{}".format(str(source[3])),
+                       "normal@{}".format(str(source[1])),
+                       "roughness@{}".format(str(source[2]))],
+            set_value=["$outputsize@{r},{r}".format(r=math.log2(env['RESOLUTION'][0]))])
```

Comment:
math.log2 is not python 2.7 compatible, so please change to int(math.log(x, 2)).
Convert to use both RESOLUTION[0] and RESOLUTION[1]